language-design.md

```diff
@@ -38,7 +69,14 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+A well-designed language is extendible, has a relatively small core vocabulary, is easy to learn (though not necessarily for everyone),
```

Comment:
So for whom would a language have to be easy to learn for it to be deemed well designed?